master-firmware/src/arms/arm.h

```diff
@@ -0,0 +1,34 @@
+#ifndef ARM_API_H
```

Comment:
Maybe we should name C++ headers files with ".hpp".

Reply:
Not sure what we should have as standard